src/dbt/powerschool/models/intermediate/int_powerschool__calendar_week.sql

```diff
@@ -1,4 +1,15 @@
 with
+    schools as (
+        select
+            _dbt_source_relation,
+            school_number,
+
+            case
+                high_grade when 12 then 'HS' when 8 then 'MS' when 4 then 'ES'
+            end as school_level,
+        from `kipptaf_powerschool.stg_powerschool__schools`
```

Comment:
make sure you use `ref` for tables

Reply:
jesus h. dhrist, what a rookie mistake. on it!